src/ext/java/org/opentripplanner/ext/transmodelapi/model/siri/et/EstimatedCallType.java

```diff
@@ -191,12 +191,20 @@ public static GraphQLObjectType create(
                               .getAlightType(((TripTimeShort) environment.getSource()).getStopIndex()) == PICKDROP_COORDINATE_WITH_DRIVER;
                     })
                     .build())
-            .field(GraphQLFieldDefinition.newFieldDefinition()
+
+            // TODO This checks both whether the Trip or the Stop is cancelled. Change after implementing DSJ
+            .field(GraphQLFieldDefinition
+                    .newFieldDefinition()
                     .name("cancellation")
                     .type(Scalars.GraphQLBoolean)
                     .description("Whether stop is cancelled.")
-                    .dataFetcher(environment -> ((TripTimeShort) environment.getSource()).isCancelledStop())
-                    .build())
+                    .dataFetcher(environment -> ((TripTimeShort) environment.getSource()).isCancelledStop()
+                        || GqlUtil.getRoutingService(environment)
+                            .getTripForId()
+                            .get(((TripTimeShort) environment.getSource()).getTripId())
+                            .getTripAlteration()
+                            .isCanceledOrReplaced())
+            .build())
```

Comment:
Burde ikke dette vært gjort inne i TripTimeShort. Regner med at de andre plassene som bruker TripTimeShort også tranger denne logikken. Evt. to methoder på TripTimeShort:
```
isCancelledStop()

/** Return true if stop is cancelled or trip is canceled or replaced */
isCanceledEffectively()
```
Sjekk bruken av isCancelledStop() om heller isCanceledEffectively() burde vært brukt.